Adyen/Core/Components/Base/BasicComponentConfiguration.swift

```diff
@@ -8,21 +8,33 @@ import Foundation
 
 /// Any component's most basic configuration.
 public protocol AnyBasicComponentConfiguration: Localizable {
-    
-    /// The form component style.
-    var style: FormComponentStyle { get }
+
+    /// The payment information.
+    var payment: Payment? { get }
```

Comment:
maybe rename `PaymentAwareComponent` to `PaymentAware` and make `AnyBasicComponentConfiguration` implement it

Reply:
👍